package.json

```diff
@@ -20,6 +20,7 @@
     "@chenfengyuan/vue-qrcode": "1.0.2",
     "axios": "^1.6.0",
     "chart.js": "2.9",
+    "fs": "^0.0.1-security",
```

Comment:
Why is this package required?

https://www.npmjs.com/package/fs:

> This package name is not currently in use, but was formerly occupied by another package. To avoid malicious use, npm is hanging on to the package name, but loosely, and we'll probably give it to you if you want it.